src/net.cpp

```diff
@@ -381,9 +381,9 @@ bool CConnman::CheckIncomingNonce(uint64_t nonce)
 }
 
 /** Get the bind address for a socket as CAddress */
```

Comment:
Still documented as returning a `CAddress`?

Reply:
Should be "CService", fixed, thanks!